### PR TITLE
New ruleset: hrl_files

### PR DESCRIPTION
**Pull request by elbrujohalcon**
As a first step towards #162 and #268, I'm creating the `hrl_files` ruleset here.
I basically copied everything from `erl_files` and removed the ones that didn't seem to make sense on hrl files.
To be fair, I would've kept all rules because _dirty people_ can do _dirty things_ on hrl files, but… well… we can improve that list over time if we want to, anyway.

### CODE REVIEW

File: test/elvis_SUITE.erl [hunk 1]
```diff
@@ -14,14 +14,10 @@
...
          rock_with_rule_groups/1, rock_this_skipping_files/1, rock_this_not_skipping_files/1,
-         rock_with_umbrella_apps/1, custom_ruleset/1, throw_configuration/1,
+         rock_with_umbrella_apps/1, custom_ruleset/1, hrl_ruleset/1, throw_configuration/1,
          find_file_and_check_src/1, find_file_with_ignore/1, invalid_file/1, to_string/1,
          chunk_fold/1]).
 
-                                 %% Rocking
-
-         %% Util & Config
-
```
Comment (elbrujohalcon) [inline test/elvis_SUITE.erl:24]: Leftovers from when we formatted the code using the formatter 🤷🏻

**Review by paulo-ferraz-oliveira — APPROVED** [review]
Would you create an issue so that the Wiki's updated later according to these changes?

**Comment by elbrujohalcon** [comment]
I'll update the wiki right now, don't worry.

**Comment by elbrujohalcon** [comment]
The wiki was updated, but I wonder if we shouldn't just close the wiki in favor of RULES.md

**Pull request merged**